bindings/rust/wasmedge-sys/src/vm.rs

```diff
@@ -430,6 +452,25 @@ impl Vm {
         self.run_function_async(func_name, params)
     }
 
+    pub async fn run_wasm_from_bytes_async2(
+        &self,
+        bytes: &[u8],
+        func_name: impl AsRef<str> + Send,
+        params: impl IntoIterator<Item = WasmValue> + Send,
+    ) -> WasmEdgeResult<Vec<WasmValue>> {
+        // load
+        self.load_wasm_from_bytes(bytes)?;
+
+        // validate
+        self.validate()?;
+
+        // instantiate
+        self.instantiate()?;
+
+        // invoke
+        self.run_function_async2(func_name, params).await
+    }
+
```

Comment:
Please replace the origin `run_wasm_from_bytes_async` with the new `run_wasm_from_bytes_async2`, but keep using the original API name `run_wasm_from_bytes_async`. Thanks!